src/lib.rs

```diff
@@ -162,48 +164,56 @@ mod tests {
     }
 
     impl Instruction for TestInstruction {
-        fn address(&self) -> Address {
+        #[allow(unused_variables)]
+        fn address(&self, disassembler: &Disassembler) -> Address {
```

Comment:
Just prefix the argument name with `_` like `_disassembler` instead of using `#[allow(unused_variables)]`.